### PR TITLE
Update tokio to 1.0.1

### PR DESCRIPTION
**Pull request by kiron1**
- _Rename with_io to try_io and drop with_poll_ - https://github.com/tokio-rs/tokio/pull/3306

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 
 name    = "inotify"
-version = "0.9.1"
+version = "0.9.2"
```
Comment (hannobraun) [inline Cargo.toml:4]: Can you please remove this change? I will update the version when I do the release.

**Review by hannobraun — CHANGES_REQUESTED** [review]
Thank you, @kiron1! I've left a minor change request.

File: Cargo.toml (hunk 1 above)
Comment (kiron1) [reply Cargo.toml]: Sure.

File: Cargo.toml [hunk 2]
```diff
@@ -35,12 +35,12 @@ bitflags     = "1"
 futures-core = { version = "0.3.1", optional = true }
 inotify-sys  = "0.1.3"
 libc         = "0.2"
-tokio        = { version = "0.3.3", optional = true, features = ["net"] }
+tokio        = { version = "1.0.0", optional = true, features = ["net"] }
```
Comment (Darksonn) [inline Cargo.toml:38]: The `1.0.0` version is yanked.
```suggestion
tokio        = { version = "1.0.1", optional = true, features = ["net"] }
```

Comment (kiron1) [reply Cargo.toml:38]: Thanks for the hint.

**Comment by hannobraun** [comment]
Thanks again, @kiron1, and thanks for the help, @Darksonn!

**Pull request merged**